order-service/src/main/resources/application-local.yml

```
@@ -50,4 +59,8 @@ resilience4j:
     configs:
       default:
         timeout-duration: 5s
-        cancel-running-future: true
\ No newline at end of file
+        cancel-running-future: true
+order:
+  cancel-policy:
+    order-pending: 1
+    payment-pending: 1
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)